### ISSUE TITLE
How to change the results image pixel?

### ISSUE THREAD

**Issue by andyandytony** [christopher-beckham/hologan-pytorch#4]

**Comment by christopher-beckham** [comment]
???

**Comment by andyandytony** [comment]
![1587696623736](https://user-images.githubusercontent.com/50736064/80367297-6b61f600-88bd-11ea-9d66-f2ca58fd36cf.png)

for example this image cars are too small and how to change the each pixels.
![Inked1587696623736_LI](https://user-images.githubusercontent.com/50736064/80367551-e2978a00-88bd-11ea-9a6e-59ef8c137d7b.jpg)

**Comment by andyandytony** [comment]
and i know how to change column amd row numbers.    but each pixels are same

**Comment by christopher-beckham** [comment]
Do you mean, how to change the resolution of the image? If so you'll need to modify the transform in your specific Dataset class as well as modify the architecture in holonet.py to be able to generate that resolution image by adding extra, say, strided transposed convolutions (and potentially also add another strided resblock to the discriminator). Neither the generator nor discriminator have arguments that let you set the desired image resolution, though I could easily make that possible in the next commit (I have planned to fix some things).

If you mean just changing the resolution of the saved images, you can use something like `torch.nn.functional.interpolate(imgs, scale_factor)` on the tensor of images before `save_image` is called: https://pytorch.org/docs/stable/nn.functional.html#interpolate

**Comment by christopher-beckham** [comment]
I will be committing some new code soon which should improve any experiments that are run (addressing the issue with the identity loss I mention in the readme). It already looks like you are getting decent car rotations (though maybe it is worth tuning the min and max angles for the y axis if you haven't already).

**Comment by andyandytony** [comment]
thank you a lots~~
